src/object.d

```diff
@@ -1974,6 +1975,11 @@ void clear(T : Value[Key], Value, Key)(T* aa)
     _aaClear(*cast(void **) aa);
 }
 
+void reserve(T : Value[Key], Value, Key)(ref T aa, size_t ndim)
+{
+    _aaReserve(cast(void **)&aa, typeid(Value[Key]), ndim);
```

Comment:
You can use `typeid(T)` here since `T == Value[Key]`.

Reply:
True, however I used Value[Key] to be consistent with the rest of the file.

Reply:
There are a few kinks we could do better in this area, si @Darredevil let's go with the cleaner way here and we'll update the rest in good time.